pajbot/modules/roulette.py

```diff
@@ -5,7 +5,6 @@
 
 import pajbot.exc
 import pajbot.models
-import pajbot.utils
 from pajbot import utils
```

Comment:
should have normalized other usages of `utils.` too, since i see the import remained here